sensor_msgs/Image im
---
int8 floorNumber
